src/api/json/catalog.json

```diff
@@ -302,7 +302,10 @@
       "name": "sarif.json",
       "description": "Static Analysis Results Interchange Format (SARIF)",
       "fileMatch": [ "*.sarif,*.sarif.json" ],
-      "url": "http://json.schemastore.org/sarif"
+      "url": "http://json.schemastore.org/sarif",
+      "versions": {
+        "1.0.0": "http://json.schemastore.org/sarif-1.0.0"
```

Comment:
Yes, that is because you uploaded the sarif-1.0.0.json file. This line just points to it from the API file